modules/understanding-upgrade-channels.adoc

```diff
@@ -86,26 +86,17 @@ You can use the `stable-{product-version}` channel to upgrade from a previous mi
-In addition to the stable channel, certain minor versions of {product-title} offer an
-link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[Extended Update Support] (EUS).
-These EUS versions extend the maintenance phase for customers with Premium Subscriptions
-to 14 months. {product-title} 4.6 is currently the only minor version with EUS.
-
-Although there is no difference between stable-4.6 and eus-4.6
-channels until {product-title} 4.6 transitions to EUS the phase, you can switch
-to the EUS channel as soon as it becomes available. When {product-title} 4.6
-transitions to the EUS phase of its lifecycle the stable-4.6 channel will
-no longer receive subsequent z-stream updates. After you upgrade to a version that
-is exclusive to the EUS channel, that cluster will no longer be eligible for minor
-version upgrades until upgrades to the next EUS version become available. The next
-planned EUS version is to be 4.10 and the upgrade to that version will require a
-serial set of version upgrades, such as from 4.6 to 4.7 to 4.8 to 4.9 to 4.10.
-
-Additionally, you may only switch to the EUS channel when your cluster is running
-a supported version of {product-title} 4.6.
-
-Finally, if you install of a 4.6 version that is exclusive to EUS, you will
-similarly not be able to upgrade to a later minor version until upgrades are provided to 4.10.
+In addition to the stable channel, certain minor versions of {product-title} offer an link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[Extended Update Support] (EUS). These EUS versions extend the maintenance phase for customers with Premium Subscriptions to 14 months. {product-title} 4.6 is currently the only minor version with EUS.
+
+Although there is no difference between stable-4.6 and eus-4.6 channels until {product-title} 4.6 transitions to the EUS phase, you can switch to the EUS channel as soon as it becomes available. When {product-title} 4.6 transitions to the EUS phase of its lifecycle, the stable-4.6 channel will no longer receive subsequent z-stream updates. After you upgrade to a version that is exclusive to the EUS channel, that cluster will no longer be eligible for minor version upgrades until upgrades to the next EUS version become available. The next planned EUS version is to be 4.10 and the upgrade to that version will require a serial set of version upgrades, such as from 4.6 to 4.7 to 4.8 to 4.9 to 4.10.
+
+Additionally, you can only switch to the EUS channel when your cluster is running a supported version of {product-title} 4.6.
+
+Finally, if you install a 4.6 version that is exclusive to EUS, you will similarly not be able to upgrade to a later minor version until upgrades are provided to 4.10.
+
+Both standard and non-EUS subscribers can access all EUS repositories and necessary RPMs (`rhel-*-eus-rpms`) to be able to support critical purposes such as debugging and building drivers.
+
+// EUS repositories are available by doing XYZ.
```

Comment:
Will there be steps that we should document for customers to access? If so, are they the same for both standard and premium subscribers?

Reply:
@bobfuru We will have a chapter on supporting out-of-tree drivers in 4.8, since this is where all this came from I wanted to dedicate a paragraph on how to enable the "right" repositories. Besides the EUS we also have the `rhocp-<ocp-version>-...` repositories that can hold some important packages.

Reply:
Oh, great! Thanks for the info, @zvonkok.  Not sure if that's something you or @ahardin-rh plan to add in a new PR for 4.8? If so, I can close this PR in favor of another PR for the chapter you are referring to, or we can keep this open and make necessary updates as needed when 4.8 work is completed.

Reply:
Let's keep this open as the ref point for the BZ and for tracking. Currently preparing the documentation and need some PM eyes first on it before going forward.

Reply:
Hi @zvonkok - I'm trying to discern whether I can merge this PR and close the corresponding BZ (https://bugzilla.redhat.com/show_bug.cgi?id=1929998), or if I should anticipate further feedback? Thanks.